scripts/scil_endpoints_map.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+import json
+import os
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+DESCRIPTION = '''
+Computes the endpoint map of a bundle. The endpoint map
+is simply a count of the number of streamlines that
+start or end in each voxel. The idea is to estimate the
+cortical areas affected by the bundle (assuming
+streamlines start/end in the cortex)
+'''
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_bundle',
+                   help='Fiber bundle filename.')
+    p.add_argument('reference',
+                   help='Reference anatomic filename.')
+    p.add_argument('endpoints_map_head',
```

Comment:
It's not really clear the next two arguments are outputs (use groups ?)

Reply:
I decided to improve the help.